.github/actions/javascript/authorChecklist/index.js

```diff
@@ -17707,7 +17707,7 @@ exports["default"] = _default;
 var _highlight = __nccwpck_require__(7654);
 var _picocolors = _interopRequireWildcard(__nccwpck_require__(7023), true);
 function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return null; var r = new WeakMap(), t = new WeakMap(); return (_getRequireWildcardCache = function (e) { return e ? t : r; })(e); }
-function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && Object.prototype.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
```

Comment:
This is a generated file that can be ignored. I guess it was affected because `npm` decided to change a version of babel used in `package-lock.json`